src/tokens/split_spaces.c

```diff
@@ -50,6 +50,16 @@ int	add_spaces(t_token_lst *const token_lst, t_token **token,
 	return (free(split), EXIT_SUCCESS);
 }
 
+/**
+ * @brief If token being expanded is unquoted and contains space,
+ * separates between as many tokens needed.
+ * Uses t_token_lst_add_after() to do that.
```

Comment:
Again, the `brief` section is not supposed to explain how the function proceeds.